cluster-up/cluster/k8s-1.18/README.md

```diff
@@ -55,3 +55,27 @@ volume contains corrupt data, it can be deleted with
 ```bash
 docker volume rm kubevirt_registry
 ```
+
+## Enabling IPv6 connectivity
+
+In order to be able to reach from the cluster to the host's IPv6 network, IPv6
+has to be enabled on your Docker. Add following to your
+`/etc/docker/daemon.json` and restart docker service:
+
+```json
+{
+    "ipv6": true,
+    "fixed-cidr-v6": "2001:db8:1::/64"
+}
+```  
```

Comment:
nit: 2 spaces at the end here, its annoying with git CLI, no need to fix in this PR of course.